server/cmd/genhooks/main.go

```diff
@@ -24,14 +24,14 @@ import (
 var (
 	hookName   string
 	urlPattern string
-	prority    int64
+	priority   int64
 	savePath   string
 )
```

Comment:
can you check if there's any typos like this? there's a PR related to the hook/priority. I don't know if there's more typos

Reply:
I have checked the PR related to `priority`, and fixed it.